mungegithub/mungers/approval-handler.go

```diff
@@ -21,14 +21,21 @@ import (
 
 	"k8s.io/contrib/mungegithub/features"
 	"k8s.io/contrib/mungegithub/github"
+	mungeComment "k8s.io/contrib/mungegithub/mungers/matchers/comment"
 	"k8s.io/contrib/mungegithub/mungers/mungerutil"
 	"k8s.io/kubernetes/pkg/util/sets"
 
+	"bytes"
+	"fmt"
 	"github.com/golang/glog"
 	goGithub "github.com/google/go-github/github"
 	"github.com/spf13/cobra"
 )
 
+const (
+	approvalNotificationName = "ApprovalNotifier"
```

Comment:
Not super fan of the name, maybe `[APPROVED-FILES]` ? But let's not bikeshed too much...